cpp/proto/arcticc/pb2/storage.proto

```diff
@@ -108,6 +108,7 @@ message VersionStoreConfig {
     message StorageFailureSimulator {
         double write_failure_prob = 40;
         double read_failure_prob = 41;
+        bool give_s3_slowdown_error_in_iter = 42;
```

Comment:
Find the name slightly confusing, I don't know what `in_iter` signifies?

Reply:
I was trying to keep it short. It is for giving s3 slowdown error in iteration. Maybe keep it in full?